examples/dgmulti_2d/elixir_navierstokes_convergence.jl

```diff
@@ -10,180 +10,226 @@ mu() = 0.01
-                                - v1_y      * v2_x         * mu_
-                                - v2_x      * v2_x         * mu_
-         - T_const * inv_rho_cubed * (        p_xx * rho   * rho
-                                      - 2.0 * p_x  * rho   * rho_x
-                                      + 2.0 * p    * rho_x * rho_x
-                                      -       p    * rho   * rho_xx ) * mu_
-    # stress tensor and temperature gradient terms from y-direction
-                                - v1_yy     * v1           * mu_
-                                - v2_xy     * v1           * mu_
-                                - v1_y      * v1_y         * mu_
-                                - v2_x      * v1_y         * mu_
-                                - 4.0 / 3.0 * v2_yy * v2   * mu_
-                                + 2.0 / 3.0 * v1_xy * v2   * mu_
-                                - 4.0 / 3.0 * v2_y  * v2_y * mu_
-                                + 2.0 / 3.0 * v1_x  * v2_y * mu_
-         - T_const * inv_rho_cubed * (        p_yy * rho   * rho
-                                      - 2.0 * p_y  * rho   * rho_y
-                                      + 2.0 * p    * rho_y * rho_y
-                                      -       p    * rho   * rho_yy ) * mu_ )
-
-  return SVector(du1, du2, du3, du4)
+    y = x[2]
+
+    # TODO: parabolic
+    # we currently need to hardcode these parameters until we fix the "combined equation" issue
+    # see also https://github.com/trixi-framework/Trixi.jl/pull/1160
+    inv_gamma_minus_one = inv(equations.gamma - 1)
+    Pr = prandtl_number()
+    mu_ = mu()
+
+    # Same settings as in `initial_condition`
+    # Amplitude and shift
+    A = 0.5
+    c = 2.0
+
+    # convenience values for trig. functions
+    pi_x = pi * x[1]
+    pi_y = pi * x[2]
+    pi_t = pi * t
+
+    # compute the manufactured solution and all necessary derivatives
+    rho = c + A * sin(pi_x) * cos(pi_y) * cos(pi_t)
+    rho_t = -pi * A * sin(pi_x) * cos(pi_y) * sin(pi_t)
+    rho_x = pi * A * cos(pi_x) * cos(pi_y) * cos(pi_t)
+    rho_y = -pi * A * sin(pi_x) * sin(pi_y) * cos(pi_t)
+    rho_xx = -pi * pi * A * sin(pi_x) * cos(pi_y) * cos(pi_t)
+    rho_yy = -pi * pi * A * sin(pi_x) * cos(pi_y) * cos(pi_t)
+
+    v1 = sin(pi_x) * log(y + 2.0) * (1.0 - exp(-A * (y - 1.0))) * cos(pi_t)
+    v1_t = -pi * sin(pi_x) * log(y + 2.0) * (1.0 - exp(-A * (y - 1.0))) * sin(pi_t)
+    v1_x = pi * cos(pi_x) * log(y + 2.0) * (1.0 - exp(-A * (y - 1.0))) * cos(pi_t)
+    v1_y = sin(pi_x) *
+           (A * log(y + 2.0) * exp(-A * (y - 1.0)) +
+            (1.0 - exp(-A * (y - 1.0))) / (y + 2.0)) * cos(pi_t)
+    v1_xx = -pi * pi * sin(pi_x) * log(y + 2.0) * (1.0 - exp(-A * (y - 1.0))) * cos(pi_t)
+    v1_xy = pi * cos(pi_x) *
+            (A * log(y + 2.0) * exp(-A * (y - 1.0)) +
+             (1.0 - exp(-A * (y - 1.0))) / (y + 2.0)) * cos(pi_t)
+    v1_yy = (sin(pi_x) *
+             (2.0 * A * exp(-A * (y - 1.0)) / (y + 2.0)
+              -
```

Comment:
These minus and plus signs in separate lines look weird and make this section overly long.

Reply:
I found an imho acceptable version in a5d572aa. Could you maybe apply something similar to the other files you found (probably all navierstokes_convergence files for any mesh type)